src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -50,7 +50,7 @@ namespace cryptonote
 // advance which version they will stop working with
 // Don't go over 32767 for any of these
 #define CORE_RPC_VERSION_MAJOR 2
-#define CORE_RPC_VERSION_MINOR 1
+#define CORE_RPC_VERSION_MINOR 2
```

Comment:
What is the minor version used for? I think older clients are still compatible, as the receiver would just assume that `get_txid == true` (existing behavior).

Reply:
It gets bumped if the changes are backward compatible. They are backward compatible here as you say. A client can now tell whether some particular thing (here, get_txid) is in the daemon it's talking to.